spec/marc_to_solr/lib/config_spec.rb

```diff
@@ -582,6 +582,15 @@ def fixture_record(fixture_name, indexer: @indexer)
         record = @indexer.map_record(MARC::Record.new_from_hash('fields' => [f650], 'leader' => leader))
         expect(record['geographic_facet'].first).to eq('France')
       end
+
+      describe 'local overrides' do
+        let(:f650) { { '650' => { 'ind1' => ' ', 'ind2' => '0', 'subfields' => [{ 'a' => 'Oil spills', 'z' => 'America, Gulf of' }] } } }
+
+        it 'creates a facet for local marc geographic locations' do
+          record = @indexer.map_record(MARC::Record.new_from_hash('fields' => [f650], 'leader' => leader))
+          expect(record['geographic_facet'].first).to eq('Mexico, Gulf of')
+        end
+      end
```

Comment:
Not a blocker, but I think it would be nice to have a test for a 651$a geographic facet as well.

Reply:
Done